masterlist.yaml

```
@@ -22679,6 +22679,9 @@ plugins:
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
       - 'Immersive Citizens - OCS patch.esp'
+    req: [ *SKSE ]
```

@markekraus You can actually utilize one of LOOT's condition functions & the fact that you put your mod's version number in the plugin's description to reflect that SKSE is only needed for v4.1.0 of your mod. This way, the requirement message will only be displayed if the user doesn't have SKSE installed & they're using v4.1.0 or greater of your mod.

```suggestion
    req:
      - <<: *SKSE
        condition: 'version("MarkekrausSentientChairsOfSkyrim.esp", "4.1.0", >=)'
```

---

awesome... I wasn't sure this would work as I didn't see any examples in the file that does that.. also... this is the first version where the header exists.. will that be a problem?

---

Files without version data are interpreted by LOOT as having a version number of 0 for evaluating the version function, so it'll work as intended.